manage.py

```diff
@@ -254,10 +254,10 @@ def create_all():
     app = create_app(parse_options())
     log = logging.getLogger(__name__)
     with app.app_context():
-        tables_before = {t[0] for t in db.session.execute('SHOW TABLES')}
+        tables_before = db.engine.table_name()
```

Comment:
Why use the same data-type of the variable and enclose it in set() here. Rather than changing like 260.